graviti/utility/common.py

```diff
@@ -67,3 +68,52 @@ def shorten(origin: str) -> str:
 
     """
     return origin[:7]
+
+
+class CheckArgs:
+    """Check the input arguments of a function.
+
+    Arguments:
+        args: The names of the arguments to be checked.
+
+    """
+
+    def __init__(self, *args: str) -> None:
+        self._names = args
+
+    def __call__(self, func: Callable[..., Any]) -> Callable[..., Any]:
```

Comment:
we need to use TypeVar as the type hint of decorator, you can find lots of examples in tensorbay